automated/linux/tvm/tvm-unit-tests.sh

```diff
@@ -0,0 +1,78 @@
+#!/bin/bash
+
+# shellcheck disable=SC1091
+. ../../lib/sh-test-lib
+OUTPUT="$(pwd)/output"
+OUTPUT_FILE="${OUTPUT}/results.txt"
+SKIP_INSTALL="false"
+TVM_TARBALL="http://snapshots.linaro.org/components/tvm/latest/tvm.tar.xz"
+TEST_DIR="/tmp/tvm-dir"
+TVM_HOME="${TEST_DIR}"/tvm
+
+usage() {
+  echo "Usage: $0 [-s <true|false>]
+                  [-t <tvm-tarball>]
+                  [-d <testing-dir-location>]
+                  [-e <tvm-home-location>]" 1>&2
+  exit 1
+}
+
+while getopts "s:t:d:e:h:" o; do
+  case "$o" in
+    s) SKIP_INSTALL="${OPTARG}" ;;
+    t) TVM_TARBALL="${OPTARG}" ;;
+    d) TEST_DIR="${OPTARG}" ;;
+    e) TVM_HOME="${OPTARG}" ;;
+    h|*) usage ;;
+  esac
+done
+
+pkgs="ntp wget systemd"
+# Package installs must be split in two steps due to installs failing from system clock being wrong.
+pkgs2="python3-dev python3-setuptools llvm"
```

Comment:
since you're not building the package, please drop python3-dev and python3-setuptools, those two should only be needed for building.